.renovaterc.json

```diff
@@ -11,11 +11,15 @@
     "@spectrum-web-components/icon",
     "@spectrum-web-components/icons",
     "@spectrum-web-components/icons-workflow",
+    "@spectrum-web-components/illustrated-message",
```

Comment:
Maybe something like `@spectrum-web-components/*` is supported?

Reply:
I think I checked that before. It's not :(